README.md

```diff
@@ -4,4 +4,64 @@ SPDX-FileCopyrightText: 2021 Benedict Harcourt <ben.harcourt@harcourtprogramming
 SPDX-License-Identifier: BSD-2-Clause
 -->
 
-# Kitteh's Bot Framework
+# MewBot
+
+MewBot is an automation framework intended to simplify building cross-platform
+text/chat-bots.
+The design is intended to be modular with configuration separated from code,
+allowing users and developers to build out custom logic and behaviours with
+minimal coding experience.
+
+### Status
+
+> :warning: This project is still in the very early stages. Some basic bots can be built
+> and run, but we currently consider all parts of the framework to be unstable.
+
+### Packages and Modules
+
+| Package                   | Modules                             | Description                                                                                                                                |
+|---------------------------|-------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------|
+| `mewbot-core`             | `mewbot.core`                       | Base interfaces for all modules                                                                                                            |
+| `mewbot-dev`              | `mewbot.component`, `mewbot.api.v1` | Development libraries + component registry system. This is the package that all 3rd party libraries should depend on for their interfaces. |
+| `mewbot-runner`           | `mewbot.loader`, `mewbot.bot`       | Tools to load a bot, and run that bot.                                                                                                     |
+| `mewbot-[discord/twitch]` | `mewbot.io.[discord/twitch]`        | The bindings to connect MewBot to a given service.                                                                                         |
+
+![module dependency graph](./mewbot.svg)
+
+### Development
+
+Contributions to the project are made via GitHub pull requests, which will
```

Comment:
Do we have a branching model?

Reply:
(do we need a branching model?)

Reply:
Those are both great questions!
The PR setup currently requires branched by up-to-date, and I would prefer to keep a semi-linear history, so that is at least some defintion. Does a branching strategy normally include things like branch naming and folder conventions?

Reply:
Two things I usually define:
branch A is "the most up to date, working, tested, and approved code"
branch B is "The version in the middle of development"
You may have A==B, or something resembling git flow, or some other construction.

Secondly, how branches for issues etc are defined, which would be naming and folder conventions.

Reply:
Makes sense! I've added a note on #9 which is for setting up contributing.md (and the code of conduct)